CommonClasses/interface/PhotonInfo.h

```diff
@@ -91,7 +91,7 @@ namespace ExoDiPhotons
     bool isMCTruthFake; // filled in analyzer
```

Comment:
The same comment that I made regarding the `diphotonInfo_t` struct applies here.

Reply:
Same here.